### PR TITLE
Add Flashcards

### PR DESCRIPTION
**Pull request by Mantas-Zilinskas**
Created a basic page for adding new flashcards to data excel file.

### CODE REVIEW

File: WebApplikacija/WebApplikacija/Controllers/ExcelController.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+using WebApplikacija.Models;
+using OfficeOpenXml;
+using Microsoft.AspNetCore.Hosting.Server;
+
+namespace WebApplikacija.Controllers
+{
+    public class ExcelController
+    {
+
+        public static void append(String filePath, Flashcard flashcard)
```
Comment (gintre) [inline WebApplikacija/WebApplikacija/Controllers/ExcelController.cs:11]: **A**ppend ? or you should change "public class Flashcard" to  **f**lashcard

**Comment by Mathild0** [comment]
ok, nice job

**Review by Mathild0 — COMMENTED** [review]
nice

**Review by Mathild0 — APPROVED** [review]
ok nice job

**Pull request merged**